media/gpt.js

```diff
@@ -242,7 +242,7 @@
                      <div class="col">
 						<img src="${icon}" class="avatar"
 						alt="Avatar" />
-						AI Guided Remediation
+						AI Security Champion
```

Comment:
Can you use the const here? 
constants.aiSecurityChampion}

Reply:
same as above